warehouse/query-core/src/main/java/datawave/query/iterator/QueryIterator.java

```diff
@@ -345,10 +345,10 @@ public void seek(Range range, Collection<ByteSequence> columnFamilies, boolean i
         // preserve the original range for use with the Final Document tracking iterator because it is placed after the ResultCountingIterator
         // so the FinalDocumentTracking iterator needs the start key with the count already appended
         originalRange = range;
-        ActiveQueryLog.getInstance().get(getQueryId()).beginCall(this.originalRange, ActiveQuery.CallType.SEEK);
+        ActiveQueryLog.getInstance(getActiveQueryLogName()).get(getQueryId()).beginCall(this.originalRange, ActiveQuery.CallType.SEEK);
```

Comment:
We should reduce unnecessary calls and cache accesses by declaring 
final ActiveQueryLog activeQueryLog = ActiveQueryLog.getInstance(getActiveQueryLogName());
and using  activeQueryLog in place of ActiveQueryLog.getInstance(getActiveQueryLogName()) in this method